src/main/java/org/jabref/JabRefGUI.java

```diff
@@ -272,6 +272,13 @@ private void setLookAndFeel() {
                     LOGGER.warn("Unable to find requested look and feel", e);
                 }
             }
+
+            // On Linux, Java FX fonts look blurry per default. This can be improved by using a non-default rendering
+            // setting. See https://github.com/woky/javafx-hates-linux
+            if (Globals.prefs.getBoolean(JabRefPreferences.FX_FONT_RENDERING_TWEAK)) {
```

Comment:
Why don't we just enable this for Linux by default then?

Reply:
Well, I think it is okay like this. Giving the Users the possibility to opt out is a good thing here, as all screens are different and maybe some users prefer the heavier font over the lighter one (to me the untweaked font looks somehow more appealing)

Reply:
hm ok.